feature/gribi/otg_tests/basic_encap_test/metadata.textproto

```diff
@@ -0,0 +1,12 @@
+platform_exceptions: {
+  platform: {
+    vendor: CISCO
+  }
+  deviations: {
+    ipv4_missing_enabled: true
+    pf_default_rule_variable_sequence_unsupported: true
+    ttl_copy_to_tunnel_header_unsupported: true
+    gribi_mac_override_with_static_arp: true
+    interface_ref_interface_id_format: true
+  }
+}
```

Comment:
Add `tags: TAGS_DATACENTER_EDGE` here and a new line

Reply:
updated